ios/RCTJShareModule/RCTJShareModule.m

```diff
@@ -143,7 +143,7 @@ - (JSHAREPlatform)getPlatformFromDic:(NSDictionary *)param {
     return;
   }
 
-  [JSHAREService getSoicalUserInfo:platform handler:^(JSHARESocialUserInfo *userInfo, NSError *error) {
+  [JSHAREService getSocialUserInfo:platform handler:^(JSHARESocialUserInfo *userInfo, NSError *error) {
```

Comment:
this issue fixed in this [commit](https://github.com/jpush/jshare-react-native/commit/00d476c86199e5a68145d91d0b10a20c5314c6e8#diff-9d9169642249f9f8e53009119bcfe9e6L174)